packages/design-system/src/styles/components/_UsaBanner.scss

```diff
@@ -17,10 +19,16 @@ $usa-banner-text-color: #1b1b1b;
   font-size: 0.79853rem;
   font-weight: 400;
```

Comment:
Same here, instead of directly copying compiled values we should be trying to build our own classes using the USWDS as a reference. These styles are from the USWDS typescale and type system, they shouldnt be just copied and used like this.

Reply:
This is also the only place in the DS that uses rem. We eventually want to use rem anyway, but if they are hardcoded here as compiled values, and if we arent using CMSDS variables, then theres no way to enforce consistency between this component and other styles.